broker/src/main/java/io/zeebe/broker/system/configuration/DataCfg.java

```diff
@@ -64,10 +67,22 @@ public int getLogIndexDensity() {
     return logIndexDensity;
   }
 
-  public void setLogIndexDensity(int logIndexDensity) {
+  public void setLogIndexDensity(final int logIndexDensity) {
     this.logIndexDensity = logIndexDensity;
   }
 
+  public boolean useMmap() {
```

Comment:
Maybe find a more neutral name. Not sure if the name is universal or particular to Atomix. At the very least it could be more verbose like "useMemoryMap()" or something like that